ocr/static/ocr/js/rbi_table.js

```diff
@@ -4,61 +1,64 @@
+const $table = $('#table')
+const $remove = $('#remove')
+let selections = []
 
-function getIdSelections() {
-  return $.map($table.bootstrapTable('getSelections'), (row) => row.id);
+function getIdSelections () {
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=8186851)
```suggestion
function getIdSelections() {
```